README.md

```diff
@@ -94,45 +94,59 @@ Architectures that are available per default (either as strings or under `qmap.A
 - `IBMQ_Bogota` (5 qubit, undirected linear chain layout)
 - `IBMQ_Casablanca` (7 qubit, undirected H-shape layout)
 - `IBMQ_Tokyo` (20 qubit, undirected brick-like layout)
+- `Rigetti_Agave` (8 qubit, bidirected ring layout)
+- `Rigetti_Aspen` (16 qubit, bidirected dumbbell layout)
 
 Whether the heuristic (*default*) or the exact mapper is used can be controlled by passing `method="heuristic"` or `method="exact"` to the `compile` function.
 
 There are several configuration options that can be passed to the `compile` function:
 
-- The heuristic mapper offers the `initial_layout` option, which allows to choose one of the following strategies for choosing an initial layout:
```

Comment:
Is there a reason for the formatting changes in the rest of this file?

Reply:
not intentional, auto format i assume